tools/WebStack.tasks.targets

```diff
@@ -178,7 +187,7 @@
 
                             if (failures > 0)
                             {
-                                foreach (XElement classWithFailure in assembly.Elements(XName.Get("class"))
+                                foreach (XElement classWithFailure in assembly.Elements(XName.Get("collection"))
```

Comment:
Tied to move from `XmlV1` to `Xml` results file. The few bits of the files we care about don't change much.